plugin/trino-opa/src/main/java/io/trino/plugin/opa/schema/TrinoColumn.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.opa.schema;
+
+import io.trino.spi.connector.SchemaTableName;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Helper record used to denote a specific column within a schema & table pair
+ * This is only used in filterColumns where simpler objects than TrinoTable help
+ * keep the code cleaner
```

Comment:
I find this helper to be confusing in the schema package.  It is really a private detail of the OpaAccessControl, and should be a private class in there.  That said, I don't think this helper helps... see my comment in OpaAccessControl